.github/workflows/build.yml

```diff
@@ -31,4 +31,52 @@ jobs:
       run: nimble simulator
     - name: Device
       working-directory: ./playdate_example
-      run: nimble device
\ No newline at end of file
+      run: nimble device
+
+  simulate:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    container: nimlang/nim
+    env:
+      HOME: /config
+    steps:
+    - name: Force non-interactive apt installations
+      run: echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+    - name: Update apt dependencies
+      run: apt-get update
+    - name: Install apt dependencies
+      run: apt-get install -y make libpng16-16 gcc-arm-none-eabi xvfb libgtk-3-0 sudo libwebkit2gtk-4.0 libwebkit2gtk-4.0-dev libsdl2-dev pulseaudio
+    - name: Setup audio sink
+      run: |
+        pulseaudio -D --exit-idle-time=-1
+        pactl load-module module-null-sink sink_name=SpeakerOutput sink_properties=device.description="Dummy_Output"
```

Comment:
Removing this will cause some audio related warnings in the simulator:

```
| ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
| ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
| ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
| ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
| ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
| ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
| ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
| ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
| ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
| ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
| ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
| ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
| ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
| ALSA lib conf.c:4732:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
| ALSA lib conf.c:5220:(snd_config_expand) Evaluate error: No such file or directory
| ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM default
```

But it looks like the tests still pass, so I'll go ahead and remove it just to cut the fat.